<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainDialog</class>
 <widget class="QDialog" name="MainDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="ImageLabel" name="resultImage">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>170</y>
     <width>140</width>
     <height>140</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../resource/resrc.qrc">:/main/result.jpg</pixmap>
   </property>
  </widget>
  <widget class="ImageLabel" name="paramImage">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>170</y>
     <width>140</width>
     <height>140</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../resource/resrc.qrc">:/main/param.jpg</pixmap>
   </property>
  </widget>
  <widget class="ImageLabel" name="processImage">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>170</y>
     <width>140</width>
     <height>140</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../resource/resrc.qrc">:/main/process.jpg</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="resultLabel">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>320</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>查看结果</string>
   </property>
  </widget>
  <widget class="QLabel" name="processLabel">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>320</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>样本处理</string>
   </property>
  </widget>
  <widget class="QLabel" name="paramLabel">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>320</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>软件设置</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ImageLabel</class>
   <extends>QLabel</extends>
   <header>component/imagelabel.h</header>
   <slots>
    <signal>released()</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resource/resrc.qrc"/>
 </resources>
 <connections/>
</ui>
